### DISCUSSION TITLE
Альтернатива CustomPage

### DISCUSSION THREAD

**Discussion by sashagm** [moonshine-software/moonshine#533] [Ideas]
Хотелось бы оставить реализации CustomPage как было в 1 версии. Указание напрямую в провайдер. Будет альтернативный способ без отдельного контроллера

 MenuItem::make(
                    'Основные настройки',
                    CustomPage::make('Основные настройки', 'setting', 'Include.Admin.setting', fn () => []),
                    'heroicons.cog-8-tooth'
                ),

**Comment by lee-to** [comment]
https://github.com/moonshine-software/moonshine/pull/531
